Userland/Libraries/LibWeb/HTML/Navigator.cpp

```diff
@@ -101,8 +102,15 @@ JS::NonnullGCPtr<UserActivation> Navigator::user_activation()
 // https://w3c.github.io/pointerevents/#dom-navigator-maxtouchpoints
 WebIDL::Long Navigator::max_touch_points()
 {
-    dbgln("FIXME: Unimplemented Navigator.maxTouchPoints");
+#if defined(AK_OS_MACOS)
+    int max_touch_points = 1;
+    return max_touch_points;
+#elif defined(AK_OS_LINUX)
+    int max_touch_points = 20;
+    return max_touch_points;
+#else
     return 0;
+#endif
```

Comment:
I don't believe it's sufficient to return an arbitrary number based on the platform like this. The number returned should depend on the underlying hardware of the given machine. For example, Chrome returns 10 on my touchscreen laptop, but 0 on my desktop. If we can't figure out what is supported by the hardware then I think the best thing to do is to return 0, which implies that touch input is not supported.